plugins/vst_base/CMakeLists.txt

```diff
@@ -31,6 +31,17 @@ BUILD_PLUGIN(vstbase vst_base.cpp VstPlugin.cpp VstPlugin.h communication.h MOCF
 
 IF(LMMS_BUILD_LINUX AND NOT WANT_VST_NOWINE)
 
+SET(WINE_INCLUDES_AND_LIBS -I/usr/include/wine/windows)
+
+# Development
+IF(EXISTS "/opt/wine-devel/lib64/libwine.so")
+	SET(WINE_INCLUDES_AND_LIBS -I/opt/wine-devel/include -L/opt/wine-devel/lib)
+ENDIF()
+# Staging
+IF(EXISTS "/opt/wine-staging/lib64/libwine.so")
+	SET(WINE_INCLUDES_AND_LIBS -I/opt/wine-staging/include -L/opt/wine-staging/lib)
+ENDIF()
```

Comment:
@karmux @jasp00 can't we calculate it in [`FindWine.cmake`](https://github.com/LMMS/lmms/blob/master/cmake/modules/FindWine.cmake#L11) then and call it something like `WINE_LIBRARY_PATH` ?

> `wine/*.h` headers cannot be found from [`wine/`]

Sounds like a bug with `FindWine.cmake`, no?